docs/examples/StyleCompositionExample.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { css } from 'emotion';
+import { jsx } from '@emotion/core';
 import Select from 'react-select';
```

Comment:
The `React` import **has** to be removed and replaced by following comment:
```jsx
/** @jsx jsx */
import { jsx } from '@emotion/core';

/* ... */
```

The `css` prop won't exist/work if the `@jsx` comment doesn't exist.

Reply:
Thanks @Rall3n.